bin/ruby-build

```diff
@@ -553,6 +553,7 @@ build_package_standard() {
   local PACKAGE_CFLAGS="${package_var_name}_CFLAGS"
 
   [ "$package_var_name" = "RUBY" ] && use_homebrew_readline || true
+  [ "$package_var_name" = "RUBY" ] && use_freebsd_pkg || true
```

Comment:
This is repeating a condition from the line before. Combining the two would be a good approach:

```sh
if [ "$package_var_name" = "RUBY" ]; then
  use_homebrew_readline || use_freebsd_pkg || true
fi
```